ctrlf.el

```diff
@@ -927,6 +967,18 @@ Interactively, select from the list of all defined styles."
   (setq ctrlf--style style)
   (setq ctrlf--last-input nil))
 
+(defun ctrlf-toggle-regexp ()
+  "Toggle CTRLF style to `regexp' or back to `literal'."
+  (interactive)
+  (setq ctrlf--style
+        (if (eq ctrlf--style 'regexp) 'literal 'regexp)))
+
+(defun ctrlf-toggle-symbol ()
+  "Toggle CTRLF style to `symbol' or back to `literal'."
+  (interactive)
+  (setq ctrlf--style
+        (if (eq ctrlf--style 'symbol) 'literal 'symbol)))
+
```

Comment:
Having separate toggles for these is a great idea. That is faster to use than always having to select from a list.

Reply:
Hey I didn't think of it! Thank you long-forgotten Emacs/Isearch developer 😆